src/controllers/messagesController.ts

```diff
@@ -31,8 +34,34 @@ export default class MessagesController {
     | IResponseErrorValidation
     | IResponseErrorNotFound
     | IResponseErrorTooManyRequests
-    | IResponseSuccessJson<PaginatedCreatedMessageWithoutContentCollection>
-  > => withUserFromRequest(req, this.messagesService.getMessagesByUser);
+    | IResponseSuccessJson<PaginatedPublicMessagesCollection>
+  > =>
+    GetMessagesParameters.decode({
```

Comment:
Why not use (this middleware)[https://github.com/pagopa/io-backend/blob/ecb79053e5f02a9c2b98accf74e16df0a03eb01c/src/controllers/servicesController.ts#L56] instead?

Reply:
Done in https://github.com/pagopa/io-backend/pull/830/commits/15434c00d547ab9ae09e3b48719e133e4cfbcb22